src/rust/grapl-observe/src/metric_reporter.rs

```diff
@@ -17,6 +18,7 @@ pub struct MetricReporter {
     to stdout; then, later, a lambda reads in these messages and writes them to Cloudwatch.
     (originally recommended in an article by Yan Cui)
     */
+    // TODO: I think I gotta mutex or arc this; two threads grab it at once and sometimes print the wrong thing!
     buffer: String,
```

Comment:
turns out my understanding of rust borrowing is not good; i thought the borrow checker might, somehow, insert locks to prevent two threads from mutating this buffer at once. I was very wrong! Turns out I was seeing some unexpected re-use of the buffer within sysmon-generator during some runs.



This will be my next task.